crates/biome_js_formatter/tests/specs/prettier/prepare_tests.cjs

```diff
@@ -1,6 +1,14 @@
-const {extractPrettierTests} = require("../../../../biome_formatter_test/src/prettier/prepare_tests");
+const { promises: fs } = require("fs");
+const {
+	extractPrettierTests,
+} = require("../../../../biome_formatter_test/src/prettier/prepare_tests.cjs");
 
 async function main() {
+	const DIRS = ["./js", "./jsx", "./typescript"];
+	for (const path of DIRS) {
+		await fs.rm(path, { recursive: true, force: true });
+	}
```

Comment:
This is just to clear out the existing tests before replacing them right?

Reply:
Yes. The directories have to be removed manually before this change.

https://github.com/biomejs/biome/blob/0b28f5f47aa968bd2511224679ae1cfbcf708fd7/crates/biome_js_formatter/tests/specs/prettier/README.md?plain=1#L26